federation/cluster/common.sh

```diff
@@ -137,7 +161,7 @@ function create-federated-api-objects {
 # FEDERATION_IMAGE_TAG: push all federated images with this tag. Used for ci testing
 function push-federated-images {
     : "${FEDERATION_PUSH_REPO_BASE?Must set FEDERATION_PUSH_REPO_BASE env var}"
-    local FEDERATION_BINARIES=${FEDERATION_BINARIES:-'federation-apiserver'}
+    local FEDERATION_BINARIES=${FEDERATION_BINARIES:-'federation-apiserver,federation-controller-manager'}
```

Comment:
+1

Some of the fixes to this PR are here - https://github.com/kubernetes/kubernetes/pull/26636/commits/2bd99740786fef727bb177df69a4d53ddf47cba2 Feel free to cherrypick the commit or just copy the changes.